projects/driver_controls/inc/control_stalk.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+// Module for interfacing with the Audi S7 control stalks
+// Requires ADS1015, GPIO expander, and event queue to be initialized
+//
+// Monitors ADS1015 + MCP23008 over I2C and raises events on input state change
+// Raises INPUT_EVENT_CONTROL_STALK_* events with empty data fields.
+//
```

Comment:
Can you add a bit about how the control stalk works (either here or maybe better in the `.c` file) regarding how the control stalk uses resistances to map outputs.